CMakeLists.txt

```diff
@@ -13,3 +14,35 @@ set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ..)
 add_executable(TransferBench src/TransferBench.cpp)
 target_include_directories(TransferBench PRIVATE src/include)
 
+find_package(ROCM 0.8 REQUIRED PATHS ${ROCM_PATH})
+include(ROCMInstallTargets)
+include(ROCMCreatePackage)
+set(ROCMCHECKS_WARN_TOOLCHAIN_VAR OFF)
+
+set(PACKAGE_NAME TB)
+set(LIBRARY_NAME TransferBench)
+
+rocm_package_setup_component(${PACKAGE_NAME}
+    LIBRARY_NAME ${LIBRARY_NAME}
+    PACKAGE_NAME ${PACKAGE_NAME}
+)
+
+# rocm_install(DIRECTORY ${PROJECT_BINARY_DIR}/src/include
+# DESTINATION include/${PACKAGE_NAME}/build
+# COMPONENT ${LIBRARY_NAME}-headers
+# FILES_MATCHING
+# PATTERN "*.hpp"
+# )
+
+rocm_install(DIRECTORY src/include
+DESTINATION include/${PACKAGE_NAME}
+COMPONENT ${LIBRARY_NAME}
+FILES_MATCHING
+PATTERN "*.hpp"
+)
+
+rocm_create_package(
+    NAME ${LIBRARY_NAME}
+    DESCRIPTION "TransferBench package"
+    MAINTAINER "RCCL Team"
```

Comment:
An email should be included in angle brackets after the maintainer name.

Reply:
Hi Lauren,
Thank you very much for the feedbacks!
I applied the modifications. Does it look correct to you?
Also, for creating the packages after "cmake .. && make", I do "make package"!
Is it the correct way to do so?
Then, I get the following error for RPM:
CPackRPM:Debug: You may consult rpmbuild logs in: 
CPackRPM:Debug:    -TransferBench-dev/build/_CPack_Packages/Linux/RPM/rpmbuildtransferbench-devel.err
CPackRPM:Debug: *** error: line 9: Illegal sequence ".." in: Version:        ..

CMake Error at /snap/cmake/1270/share/cmake-3.26/Modules/Internal/CPack/CPackRPM.cmake:1897 (message):
  RPM package was not generated!
  TransferBench-dev/build/_CPack_Packages/Linux/RPM
Call Stack (most recent call first):
  /snap/cmake/1270/share/cmake-3.26/Modules/Internal/CPack/CPackRPM.cmake:1968 (cpack_rpm_generate_package)

CPack Error: Error while execution CPackRPM.cmake

Reply:
I think the problem was VERSION in the project command!
It is solved now and seems to be working!